docs/calico-with-docker/docker-network-plugin/CalicoAsService.md

```diff
@@ -0,0 +1,141 @@
+The `calicoctl node` command already starts calico as its own container process
+on your system. There is no need to run the base `calico/node` container with a
+different command.  This section provides an example for running the `calicoctl 
+node` command with required components in a Systemd service.
+
+
+Included here is an EnvironmentFile that defines the Environment variables for
+Calico and a sample systemd service file that uses the environment file and
+starts the Calico node image as a service.
+
+`calico.env` - the EnvironmentFile:
+```
+ETCD_AUTHORITY=localhost:2379
+ETCD_SCHEME=http
+ETCD_CA_FILE=""
+ETCD_CERT_FILE=""
+ETCD_KEY_FILE=""
+```
+
+Be sure to update this environment file as necessary, such as modifying the
+ETCD_AUTHORITY value to point at the correct instance of Etcd.
+
+>Note: The ETCD_SCHEME, ETCD_CA_FILE, ETCD_CERT_FILE, and ETCD_KEY_FILE
+>environment variables are required when using Etcd with SSL/TLS.  The values
+>here are standard values for a non-SSL version of Etcd, but you can use this
+>template to define your SSL values if desired.  For more details about running 
+> Calico with Etcd using SSL/TLS, check out the
+> [Etcd Secure Cluster guide](../../EtcdSecureCluster.md).
+
+`calico-node.service` - the Systemd service:
+
+```
+[Unit]
+Description=calico-node
+After=docker.service
+Requires=docker.service
+
+[Service]
+EnvironmentFile=/etc/calico/calico.env
+ExecStartPre=-/usr/bin/docker rm -f calico-node
+ExecStart=/usr/bin/calicoctl node
+ExecStop=-/usr/bin/docker stop calico-node
+
+[Install]
+WantedBy=multi-user.target
+```
+
+The Systemd service above does the following on start:
+  - Confirm docker is installed under the `[Unit]` section
+  - Get environment variables from the environment file above
+  - Remove existing `calico-node` container (if it exists)
+  - Start `calico/node`
+
+The script will also stop the calico-node container when the service is stopped.
+
+**Note**: Depending on how you've installed Docker, the name of the Docker service
+under the `[Unit]` section may be different (such as `docker-engine.service`).
+Be sure to check this before starting the service.
+
+## Starting `calico/node-libnetwork` on its own
+If you are running Calico as a Docker network plugin, it is also necessary to
+have a separate job for handling the Calico libnetwork driver.
+
+You can start the `calico/node-libnetwork` image directly with the `docker run`
+command:
+
+```
+docker run -d --privileged --net=host \
+    -v /run/docker/plugins:/run/docker/plugins \
+    --name=calico-libnetwork -e ETCD_AUTHORITY=localhost:2379
+    calico/node-libnetwork
+```
+
+In this example,
+
+ - Privileged is required since the container creates network devices.
+ - Host network is used since the network changes need to occur in the host namespace
+ - The /run/docker/plugins volume is used to allow the plugin to communicate with Docker.
+
+### Systemd service example
```

Comment:
I like having this split out as a separate sub-section, but you didn't do this for the calico/node